x-pack/test/alerting_api_integration/security_and_spaces/tests/actions/builtin_action_types/email.ts

```diff
@@ -11,11 +11,8 @@ import { FtrProviderContext } from '../../../../common/ftr_provider_context';
 // eslint-disable-next-line import/no-default-export
 export default function emailTest({ getService }: FtrProviderContext) {
   const supertest = getService('supertest');
-  const esArchiver = getService('esArchiver');
```

Comment:
File this under "how did this ever work?!".  `esArchiver.unload('empty_kibana'))` will delete any existing spaces within Kibana when it runs.  We have a bunch of test modules that used this on **their own** `after()` processing, but the spaces are created in [an outer `describe()` section with it's own `after()` section that calls `esArchiver.unload()`](https://github.com/elastic/kibana/blob/master/x-pack/test/alerting_api_integration/security_and_spaces/tests/index.ts).  Which means that any tests run after one of these modules did the unload, would likely fail because the spaces no longer existed.  But the tests didn't fail.  No idea why.  Maybe actions/alerts isn't "touching" the same spaces APIs that event log does (my guess).  But the new tests in this PR did fail.  So I ended removing all these calls in the "inner" module tests.  Everything still passes!

Also note, `esArchiver.unload()` is an async method, and most/all of our usage didn't wait on it, so ... comedy of errors there :-)